### PR TITLE
Back to universal links

### PR DESCRIPTION
**Pull request by biafra23**
Changes proposed in this pull request:
- This change removes the code that uses URL schemes and reverts to the state where we use universal links

### CODE REVIEW

File: Multisig/UI/Safe Management/Create Safe/GoogleWeb3AuthLoginModel.swift [hunk 1]
```diff
@@ -3,46 +3,42 @@ import CustomAuth
...
-    init(authorizationComplete: @escaping () -> Void, keyGenerationComplete: @escaping ((_ key: String, _ email: String?) -> Void)) {
-        self.authorizationComplete = authorizationComplete
-        self.keyGenerationComplete = keyGenerationComplete
+    init(onClose: @escaping ((_ key: String?, _ email: String?) -> Void)) {
+        self.onClose = onClose
     }
 
     static func handle(url: URL) -> Bool {
-        if url.absoluteString.starts(with: App.configuration.web3auth.redirectScheme + "://") {
+        LogService.shared.debug("----> handle(url: \(url.absoluteString))")
```
Comment (DmitryBespalov) [inline Multisig/UI/Safe Management/Create Safe/GoogleWeb3AuthLoginModel.swift:13]: is this line still needed?

Comment (biafra23) [reply Multisig/UI/Safe Management/Create Safe/GoogleWeb3AuthLoginModel.swift:13]: At least until we figured out how to implement the html page, yes.

File: Multisig/Info.plist [hunk 2]
```diff
@@ -25,19 +27,7 @@
...
-			<key>CFBundleURLName</key>
-			<string>safe.global.app</string>
-			<key>CFBundleURLSchemes</key>
-			<array>
-				<string>$(WEB3AUTH_REDIRECT_SCHEME)</string>
-			</array>
-		</dict>
-		<dict/>
-	</array>
+	<array/>
```
Comment (MouazAlzahabi) [inline Multisig/Info.plist]: Is this close tag needed ?

File: Multisig/Cross-layer/Configuration/Config.Example.xcconfig [hunk 3]
```diff
@@ -186,7 +186,10 @@ WEB3AUTH_GOOGLE_CLIENT_ID = YOUR_WEB3AUTH_GOOGLE_CLIENT_ID
 WEB3AUTH_GOOGLE_VERIFIER = YOUR_WEB3AUTH_GOOGLE_VERIFIER
 // Use value from ios OAuth client. Credentials -> Safe{Wallet} (type ios) ->  Additional Information -> iOS URL scheme
 // Do not add :/$()/ at the end. It won't pass the build stage
-WEB3AUTH_REDIRECT_SCHEME = YOUR_WEB3AUTH_REDIRECT_SCHEME
+_WEB3AUTH_REDIRECT_URL_DEV = https:/$()/safe-wallet-web.dev.5afe.dev/web3auth/
```
Comment (MouazAlzahabi) [inline Multisig/Cross-layer/Configuration/Config.Example.xcconfig:186]: By doing this we don't use the value from Jenkins file is it intended ?

Comment (biafra23) [reply Multisig/Cross-layer/Configuration/Config.Example.xcconfig:186]: The redirect urls are not secret. We do not need to take them from jenkins.

File: Multisig/Info.plist (hunk 2 above)
Comment (biafra23) [reply Multisig/Info.plist]: I don't know. I wanted to remove these two: 
image

Comment (biafra23) [reply Multisig/Info.plist]: It's gone now.

File: Multisig/Cross-layer/Configuration/Config.Example.xcconfig [hunk 4]
```diff
@@ -180,13 +180,22 @@ ASSETCATALOG_COMPILER_APPICON_NAME  = AppIcon_$(SERVICE_ENV)
 // Also, Firebase Analytics requires clients to add -ObjC linker option.
 OTHER_LDFLAGS = $(inherited) -fprofile-instr-generate -ObjC
 
-// Set in https://console.cloud.google.com/apis/credentials?project=web3auth-safe&supportedpurview=project
-WEB3AUTH_GOOGLE_CLIENT_ID = YOUR_WEB3AUTH_GOOGLE_CLIENT_ID
+// Set in https://console.cloud.google.com/apis/credentials
+_WEB3AUTH_GOOGLE_CLIENT_ID_DEV = YOUR_WEB3AUTH_GOOGLE_CLIENT_ID_DEV
```
Comment (MouazAlzahabi) [inline Multisig/Cross-layer/Configuration/Config.Example.xcconfig]: This change for Client ID is wrong, just revert it

File: Multisig/Cross-layer/Configuration/Config.Example.xcconfig [hunk 5]
```diff
@@ -180,13 +180,22 @@ ASSETCATALOG_COMPILER_APPICON_NAME  = AppIcon_$(SERVICE_ENV)
...
+_WEB3AUTH_REDIRECT_URL_STAGING = https:/$()/safe-wallet-web.staging.5afe.dev/web3auth/
+_WEB3AUTH_REDIRECT_URL_PROD = https:/$()/app.safe.global/web3auth/
+WEB3AUTH_REDIRECT_URL = $(_WEB3AUTH_REDIRECT_URL_$(SERVICE_ENV):default=$(_WEB3AUTH_REDIRECT_URL_PROD))
+
 // Set in https://dashboard.web3auth.io/home/customauth
-WEB3AUTH_GOOGLE_VERIFIER = YOUR_WEB3AUTH_GOOGLE_VERIFIER
-// Use value from ios OAuth client. Credentials -> Safe{Wallet} (type ios) ->  Additional Information -> iOS URL scheme
-// Do not add :/$()/ at the end. It won't pass the build stage
-WEB3AUTH_REDIRECT_SCHEME = YOUR_WEB3AUTH_REDIRECT_SCHEME
+_WEB3AUTH_GOOGLE_VERIFIER_DEV = YOUR_WEB3AUTH_GOOGLE_VERIFIER_DEV
```
Comment (MouazAlzahabi) [inline Multisig/Cross-layer/Configuration/Config.Example.xcconfig]: This change for verifier is wrong, just revert it

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request merged**